### DISCUSSION TITLE
Laravel Backup Requires Extension "ext-pcntl" on Windows When Upgrading to Version 8.0

### DISCUSSION THREAD

**Discussion by aryno** [spatie/laravel-backup#1479] [Q&A]
I was recently updating my Laravel application to V9 on PHP 8.0.10 on  Windows 10 and got this error message when running composer update saying 
"spatie/laravel-backup[8.0.0, ..., 8.0.1] require ext-pcntl * -> it is missing from your system. Install or enable PHP's pcntl extension.".

Laravel backup is one of the core dependencies but the extension ext-pcntl seems to be unavailable for the Windows OS.

**Comment by freekmurze** [comment]
Our package does not officially supports windows. It might work for the most part without that extension though. 

You could instruction composer to ignore those requirements, but I don't know the flag by heart, look that up at the composer site.

**Reply by erikn69** [reply]
On V7 `ext-pcntl` was optional with `SignalRegistry::isSupported()` #1352, Why can't V8 stay that way?
https://github.com/spatie/laravel-backup/blob/ce99f8abaf4856100f12d1279e83e36f952908ac/src/Commands/BaseCommand.php#L17-L19
`composer require "spatie/laravel-backup:^8.0" --ignore-platform-reqs`
Using the flag `--ignore-platform-reqs` makes ignore platform requirements for all the packages when updating

**Reply by freekmurze** [reply]
I'd accept a PR that adds the necessary changes to v8

**Reply by erikn69** [reply]
PR #1481
